frontend/src/lib/components/cookie_disclaimer.svelte

```diff
@@ -1,9 +1,41 @@
 <script lang="ts">
+  import {
+    cookieDisclaimer,
+    allowedCookies,
+    allowNecessaryCookies,
+  } from "$lib/stores/cookies"
+  import type { CookieType } from "$lib/stores/cookies"
   import { fade, fly } from "svelte/transition"
-  import { cookieDisclaimer } from "$lib/stores/cookies"
   import { onMount } from "svelte"
+  import { browser } from "$app/environment"
+  import { cookieSelection, necessary, preferences, statistical } from "$lib/cookies"
 
-  let loadDisclaimer = false
+  const cookieDisclaimerTabs = ["Consent", "Options"]
+
+  let currentTab = 0
+  let loadDisclaimer: boolean = true
```

Comment:
Do we need to specify boolean? Since we already set it to true, just to keep it consistent with other places

Reply:
Changed